ccmlib/scylla_node.py

```diff
@@ -1347,36 +1348,40 @@ def watch_rest_for_alive(self, nodes, timeout=120):
         instead of the log file and waits for the node to be really useable,
         not just "UP" (see issue #461)
         """
-        tofind = nodes if isinstance(nodes, list) else [nodes]
-        tofind = set([node.address() for node in tofind])
-        url_live = f"http://{self.address()}:10000/gossiper/endpoint/live"
-        url_joining = f"http://{self.address()}:10000/storage_service/nodes/joining"
-        url_tokens = f"http://{self.address()}:10000/storage_service/tokens/"
-        endtime = time.time() + timeout
-        while time.time() < endtime:
-            live = set()
-            response = requests.get(url=url_live)
-            if response.text:
-                live = set(response.json())
-            response = requests.get(url=url_joining)
-            if response.text:
-                live = live - set(response.json())
-            # Verify that node knows not only about the existance of the
-            # other node, but also its tokens:
-            if tofind.issubset(live):
-                # This node thinks that all given nodes are alive and not
-                # "joining", we're almost done, but still need to verify
-                # that the node knows the others' tokens.
-                check = tofind
-                tofind = set()
-                for n in check:
-                    response = requests.get(url=url_tokens+n)
-                    if response.text == '[]':
-                        tofind.add(n)
-            if not tofind:
-                return
-            time.sleep(0.1)
-        raise TimeoutError(f"watch_rest_for_alive() timeout after {timeout} seconds")
+        logging.getLogger('urllib3.connectionpool').disabled = True
```

Comment:
Looks good, but may I ask what was the intention of enabling urrlib3 logging in the first place?
Doesn't it always do EXACTLY what happened here - that if you do a lot of REST API requests you just log all of them?  Isn't it always a bad idea, not just in this function?

Who actually enabled this connection logging, was it CCM or dtest?

Reply:
We log everything up to debug level into dtest log file.

We disable some things by level in dtest.

But those specific logs are very very useful when doing calls to http APIs, and we have quite a few of those in dtest, I don't want to disable them all.